mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SmartTestingMavenConfigurer.java

```diff
@@ -69,6 +80,12 @@ public void afterSessionEnd(MavenSession session) throws MavenExecutionException
             return;
         }
 
+        if (enableDebugLogLevel()) {
```

Comment:
having `enableMavenDebugLogLevel(true)` for setting the level up and `enableDebugLogLevel()` for figuring out if it is set is very confusing to my taste. Please, change the second one - something like: `isDebugLogLevelEnabled`